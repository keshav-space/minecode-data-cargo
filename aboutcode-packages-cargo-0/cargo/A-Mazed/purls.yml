- pkg:cargo/A-Mazed@0.1.0
