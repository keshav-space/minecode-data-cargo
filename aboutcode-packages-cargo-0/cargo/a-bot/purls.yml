- pkg:cargo/a-bot@0.0.1
- pkg:cargo/a-bot@0.1.0
- pkg:cargo/a-bot@0.1.1
