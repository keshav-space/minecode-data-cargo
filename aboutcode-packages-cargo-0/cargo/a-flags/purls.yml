- pkg:cargo/a-flags@0.1.0
- pkg:cargo/a-flags@0.1.1
- pkg:cargo/a-flags@0.1.2
- pkg:cargo/a-flags@0.1.3
- pkg:cargo/a-flags@0.2.0
