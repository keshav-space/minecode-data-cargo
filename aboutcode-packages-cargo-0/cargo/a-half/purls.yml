- pkg:cargo/a-half@0.1.0
