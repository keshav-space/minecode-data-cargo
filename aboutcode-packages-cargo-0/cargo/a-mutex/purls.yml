- pkg:cargo/a-mutex@0.1.0
- pkg:cargo/a-mutex@0.1.1
- pkg:cargo/a-mutex@0.1.2
