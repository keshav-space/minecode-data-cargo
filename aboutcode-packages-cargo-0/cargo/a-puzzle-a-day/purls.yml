- pkg:cargo/a-puzzle-a-day@0.1.0
- pkg:cargo/a-puzzle-a-day@0.1.1
