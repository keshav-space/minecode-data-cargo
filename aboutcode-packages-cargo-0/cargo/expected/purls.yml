- pkg:cargo/expected@0.0.1
